packages/trader/src/AppV2/Utils/helper.ts

```diff
@@ -0,0 +1,26 @@
+import { formatDuration, getDiffDuration } from '@deriv/shared';
+
+type GetCardLabels = () => {
+    DAY: string;
+    DAYS: string;
+};
+
+type TRemainingTimeProps = {
+    end_time?: number;
+    start_time: moment.Moment;
+    format?: string;
+    getCardLabels: GetCardLabels;
+};
+
+export const getRemainingTime = ({ end_time, format, getCardLabels, start_time }: TRemainingTimeProps) => {
+    if (!end_time || start_time.unix() > +end_time) {
+        return;
+    }
+
+    const { days, timestamp } = formatDuration(getDiffDuration(start_time.unix(), end_time), format);
+    let remaining_time = timestamp;
+    if (days > 0) {
+        remaining_time = `${days} ${days > 1 ? getCardLabels().DAYS : getCardLabels().DAY} ${timestamp}`;
+    }
+    return remaining_time;
+};
```

Comment:
this file can be removed, please check out the explanation here: https://github.com/binary-com/deriv-app/pull/15246/files#r1638068339